config/locales/hyrax.en.yml

```diff
@@ -574,11 +576,14 @@ en:
         action:
           add_collection_to_collection: "Add collection to another collection"
           add_to_collection:        "Add to Collection"
+          admin_set_confirmation:  "Deleting this administrative set from is permanent. Click OK to delete this administrative set, or Cancel to cancel this operation"
```

Comment:
Typo in... "Deleting this administrative set from is permanent."  
Should be?...  ""Deleting this administrative set from **the repository** is permanent."

Reply:
There should be a check before adding the delete action.  The modal for delete should either show the above message with the option to delete or cancel OR it should show a message "The administrative set is not empty.  Remove all items before deleting."

Checking on actual wording of the can't delete modal.

Reply:
I think you can leave the message for now.  The delete action and modal messages have a separate issue #1507.  I would just fix the typo so it makes sense as it is right now.